examples/units_and_coordinates/radec_to_hpc_map.py

```diff
@@ -0,0 +1,134 @@
+header information is then used to make a new header with the information in
+Helioprojective and a `~sunpy.map.Map` is made.
+
+The LOFAR example file has a WCS in celestial coordinates i.e. Right Ascension and
+Declination (RA-DEC). For many solar studies we may want to plot this data in some
+Sun-centered coordinate frame, such as Helioprojective. In this example we read the
+data and header information from the LOFAR FITS file and then create a new header
+with updated WCS information to create a `~sunpy.map.Map` with a HPC coordinate frame.
+We will make use of the `astropy.coordinates` and `sunpy.coordinates` submodules
+together with `sunpy.map.make_fitswcs_header` to create a new header and generate
+a `sunpy.map.Map`.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from astropy import units as u
+from astropy.coordinates import EarthLocation, SkyCoord
+from astropy.io import fits
+from astropy.time import Time
+
+import sunpy.data.sample
+import sunpy.map
+from sunpy.coordinates import frames, sun
+
+##############################################################################
+# We will first begin be reading in the header and data from the FITS file.
+hdu = fits.open(sunpy.data.sample.LOFAR_IMAGE)
+header = hdu[0].header
+data = np.squeeze(hdu[0].data)
+
+###############################################################################
+# We can inspect the header, for example, we can print the coordinate system and
+# type projection, which here is RA-DEC.
+print(header['ctype1'], header['ctype2'])
+
+###############################################################################
+# Lets pull out the observation time and wavelength from the header, we will use
+# these to create our new header.
+obstime = Time(header['date-obs'])
+frequency = header['crval3']*u.Hz
+
+###############################################################################
+# To create a new `~sunpy.map.Map` header we need convert the reference coordinate
+# in RA-DEC to Helioprojective. To do this we will first create an `astropy.coordinates.SkyCoord`
+# of the reference coordinate from the header information. LOFAR observation are in
+# the J2000 epoch (this information is in the header).
+reference_coord = SkyCoord(header['crval1']*u.deg, header['crval2']*u.deg,
+                           frame='gcrs',
+                           obstime=obstime,
+                           distance=sun.earth_distance(obstime),
+                           equinox='J2000')
+
+###########################################################################
+# To convert the reference coordinate to Helioprojective we need the location of the
+# observer (i.e. where the observation was taken). We first establish the location on
+# Earth from which the observation takes place. We can convert this to a SkyCoord at the
+# observation time.
+lofar_loc = EarthLocation(lat=52.905329712*u.deg, lon=6.867996528*u.deg)
```

Comment:
Is it possible to get this info from the header?

Reply:
it really should be but its not in the header output information :(